packages/sit-onyx/src/components/OnyxIcon/OnyxIcon.stories.ts

```diff
@@ -0,0 +1,100 @@
+import { ONYX_COLORS, type OnyxIconProps } from "@/index";
+import happyIcon from "@sit-onyx/icons/emoji-happy-2.svg?raw";
+import { defineStorybookActionsAndVModels } from "@sit-onyx/storybook-utils";
+import type { Meta, StoryContext, StoryObj } from "@storybook/vue3";
+import OnyxIcon from "./OnyxIcon.vue";
+import { ICON_SIZES } from "./types";
+
+const ALL_ICONS = import.meta.glob("../../../node_modules/@sit-onyx/icons/src/assets/*.svg", {
+  as: "raw",
+  eager: true,
+});
+
+/**
+ * Mapping between icon SVG content (key) and icon name (value).
+ * Needed to display a labelled dropdown list of all available icons.
+ */
+const iconLabels = Object.entries(ALL_ICONS).reduce<Record<string, string>>(
```

Comment:
could the function to get iconlabels be something that the icon package exports as a feature? what we learned from CoreUI is that many more components might need an icon selection in the storybook files. this logic also is more focusing into the structure how the icons package is implemented, not sure this is a good place here for this logic (when you work on the internal implementation -> structure of packages/icons, you probably don't expect side effects in the other packages)

Reply:
-> does not have to happen in this PR! just wanted to ask for your opinion :)

Reply:
Good idea, I do not want to mix up the icon package with Storybook. That's why we have the Storybook utils package where we could move this :)